LiveSplit.AutoSplitters.xml

```diff
@@ -8856,10 +8856,10 @@
             <Game>Alien: Isolation</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/Cliffs666/ASL/master/Alien_Isolation_Auto.asl</URL>
+            <URL>https://raw.githubusercontent.com/GlorpVT/Alien-Isolation-Autosplitter/refs/heads/master/Consolidated_Splitter%20v1.2.4.asl</URL>
```

Comment:
Are you planning on adjusting the version in the file name? If so, you would have to submit a PR here every time a change happens. I recommend not doing it this way.